src/stream.cpp

```diff
@@ -303,6 +303,14 @@ namespace oxen::quic
         return std::make_pair(std::move(it), offset);
     }
 
+    void Stream::revert_stream()
+    {
+        assert(endpoint.in_event_loop());
+        log::trace(log_cat, "Stream (ID:{}) reverting after early data rejected...", _stream_id);
+        _unacked_size = 0;
```

Comment:
I left another comment about this, but it seems strange to me to only reset the unacked size, but not the unsent size: doesn't that mean that there will be an unwanted delay before we naturally try resending the data?

Reply:
Chatted about this awkwardness a few months ago I believe: There is no unsent size, stream calculates `::unsent()` using `size() - unacked()`, so setting unacked to 0 accomplished the same thing